backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -77,6 +77,11 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 				.map(Map.Entry::getValue).collect(Collectors.toList());
 	}
 
+	public long getWaitingTime() {
+		long waitingTime = 2L;
```

Comment:
Shall we declare this as a static constant in the beginning and return that instead of having a local variable?

Reply:
Done.